meta-mender-core/classes/mender-bootimg.bbclass

```diff
@@ -0,0 +1,11 @@
+# Class to create the "bootimg" type, which contains the boot partition as a raw
+# filesystem.
+
+IMAGE_CMD_bootimg() {
+    dd if=/dev/zero of="${WORKDIR}/boot.${ARTIFACTIMG_FSTYPE}" count=0 bs=1M seek=${MENDER_BOOT_PART_SIZE_MB}
+    mkfs.${ARTIFACTIMG_FSTYPE} -F "${WORKDIR}/boot.${ARTIFACTIMG_FSTYPE}" -d "${IMAGE_ROOTFS}/boot" -L boot
+    install -m 0644 "${WORKDIR}/boot.${ARTIFACTIMG_FSTYPE}" "${IMGDEPLOYDIR}/${IMAGE_NAME}.bootimg"
```

Comment:
Should we take `${MENDER_BOOT_PART_SIZE_MB} = 0` into account here, and make nothing?